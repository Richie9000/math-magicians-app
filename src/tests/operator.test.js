import operate from '../components/logic/operator';

test('test sum of 2 + 2', () => {
  expect(operate(2, 2, '+')).toBe('4');
});

test('test subtraction of 9 - 5', () => {
  expect(operate(9, 5, '-')).toBe('4');
});

test('test multiplication of 11 x 2', () => {
  expect(operate(11, 2, 'x')).toBe('22');
});

test('test division of 25 / 5', () => {
  expect(operate(25, 5, 'รท')).toBe('5');
});

test('operate snapshot test', () => {
  const calc = operate(1, 1, '+');
  expect(calc).toMatchSnapshot();
});
