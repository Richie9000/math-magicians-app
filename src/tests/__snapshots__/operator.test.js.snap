// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`operate snapshot test 1`] = `"2"`;
